src/Console/Command.php

```diff
@@ -88,6 +88,8 @@ protected function createRunner(): TestRunner
          */
         $this->arguments = AddsDefaults::to($this->arguments);
 
+        LoadStructure::in($this->testSuite->rootPath);
```

Comment:
This basically autoloads the helpers and pest file before the test suite itself.